schema.graphql

```diff
@@ -1,8 +1,95 @@
+type Account @entity {
+    accountId: ID!
+    address: ID!
+    daoAvatarAddress: ID!
+    reputation: BigInt
+    tokens: BigInt
+    stakes: [Stake!] @derivedFrom(field: "accountId")
+    votes: [Vote!] @derivedFrom(field: "accountId")
+    proposals: [Proposal!] @derivedFrom(field: "proposer")
+    redemptions: [Redemption] @derivedFrom(field: "accountId")
+}
+
+type DAO @entity {
+    avatarAddress: ID!
```

Comment:
does address should not be bytes ?

Reply:
Avatar address can be uses to uniquely identify the DAO hence I used ID.